public/app/features/dashboard/components/GenAI/utils.test.ts

```diff
@@ -35,31 +35,63 @@ describe('getDashboardChanges', () => {
 
     // Call the function to test
     const result = getDashboardChanges(dashboard);
-
     // Assertions
-    expect(result.userChanges).toEqual({
-      panels: [
-        {
-          op: 'replace',
-          originalValue: 'Panel 1',
-          value: 'New title',
-          startLineNumber: expect.any(Number),
-          path: ['panels', '0', 'title'],
-        },
-      ],
-    });
+    expect(result.migrationChanges).toEqual(
```

Comment:
@SandersAaronD I updated this test with the strings generated with the new version. The property sorting is part of the diff as well